5calls/app/src/main/java/org/a5calls/android/a5calls/controller/MainActivity.java

```diff
@@ -609,6 +619,13 @@ private void loadStats() {
         }
     }
 
+    private void loadStarredIssues() {
+        starredIssues = AppSingleton.getInstance(getApplicationContext())
+                .getDatabaseHelper().getStarredIssues();
+        Log.d(TAG, starredIssues.toString());
+        mIssuesAdapter.setStarredIssues(starredIssues);
```

Comment:
In the DBHelper file, I suggested:

> Can we instead say getStarredIssues(List<String> ids) and have the DB lookup just look through the starred ones?

I suggest that getting starred issues from an existing list might also be handy because it means we won't have to call notifyDataSetChanged, instead we can just update our mIssues and notifyItemChanged for each issue that is starred.

In other words, this class will do a lot less work and we'll won't need to redraw the issues list a second time.